Python/Query_Script/query_script.py

```diff
@@ -0,0 +1,24 @@
+from googlesearch import search
+
+def google_search_results(keyword, domain= 'com'):
+	#generator function
+	for links in search(keyword, tld= domain, pause= 2):
+		yield links
+		
+def main():
+	query = input("Enter your search keyword: ")
+	number_of_results = int(input("Enter number of links you want: "))
+	count =1
+	for link in google_search_results(query):
+		# final result is printed in the form [rank link]
+		if count> number_of_results:
```

Comment:
Use proper formatting. Run your Code through `PyLint` to find issues in your Code and fix them up.

Reply:
![check](https://user-images.githubusercontent.com/32201319/103338513-8c936b80-4a76-11eb-8e02-6d8e5750da71.png)

Reply:
> Are the dependencies really that huge?
> 
> I see that you are using just `googlesearch` for the purpose. You can use `pip show` to better take a look at the dependencies.

Sorry for that issue I have updated it, now it has only 4 dependencies